src/util/system.h

```diff
@@ -37,7 +37,6 @@
 //Dash only features
 
 extern bool fMasternodeMode;
-extern bool fDisableGovernance;
```

Comment:
Probably can drop system.h from `src/masternode/payments.cpp`

Reply:
Resolved in latest push!